src/styles/renderer/_histogram.scss

```
@@ -80,6 +80,8 @@ span.#{$lu_css_prefix}-mapping-hint {
   text-align: center;
 }
 
+$lu-drag-hint-width: 7em;
```

```suggestion
$lu_drag_hint_width: 7em !default;
```
I think we should follow the naming convension of the variables in __vars.scss_ and use underscores and add `!default` to make setting and overriding easier. Maybe we should even move the variable to the __vars.scss_ file? What do you think @sgratzl?